app/build.gradle

```diff
@@ -203,5 +203,8 @@ dependencies {
     debugImplementation "androidx.compose.ui:ui-tooling:$rootProject.composeVersion"
     implementation "androidx.compose.material3:material3:$rootProject.materialVersion"
     implementation "androidx.lifecycle:lifecycle-viewmodel-compose:$rootProject.lifecycleVersion"
+
+    implementation 'com.facebook.shimmer:shimmer:0.5.0'
```

Comment:
add the version to ext block and add a comment that will describe the dependency

Reply:
you still havent implemented this change @kmanikanta335